cldf-metadata.json

```diff
@@ -0,0 +1,298 @@
+                    },
+                    {
+                        "name": "TYPE",
+                        "datatype": {
+                            "base": "string",
+                            "format": "consonant|vowel|tone"
+                        }
+                    },
+                    {
+                        "name": "FEATURE",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "VALUE",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    }
+                ],
+                "primaryKey": [
+                    "ID"
+                ]
+            }
+        },
+        {
+            "url": "data/graphemes.tsv",
+            "tableSchema": {
+                "columns": [
+                    {
+                        "name": "PK",
+                        "datatype": {
+                            "base": "integer"
+                        }
+                    },
+                    {
+                        "name": "GRAPHEME",
+                        "dc:description": "Grapheme used in a particular transcription to denote a sound",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "NAME",
+                        "dc:description": "The ordered concatenation of feature values of the denoted sound",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "BIPA",
+                        "dc:description": "The grapheme for the denoted sound in the Broad IPA transcription system",
+                        "datatype": {
+                            "base": "string"
+                        }
+                    },
+                    {
+                        "name": "DATASET",
+                        "dc:description": "Links to the source of this grapheme"
+                    },
+                    {
+                        "name": "FREQUENCY",
+                        "datatype": {
+                            "base": "integer"
+                        }
+                    },
+                    {
+                        "name": "URL",
+                        "dc:description": "URL of the grapheme in its source online database",
+                        "datatype": {
+                            "base": "anyURI"
+                        }
+                    },
+                    {
+                        "name": "IMAGE",
+                        "dc:description": "Image of the typeset grapheme.",
+                        "valueUrl": "http://web.uvic.ca/ling/resources/ipa/charts/IPAlab/images/{IMAGE}",
```

Comment:
@LinguList if this is the location for IPA glyphs you had in mind?

Reply:
Yes. This is the location of the IPA glyphs.